src/ape/exceptions.py

```diff
@@ -139,6 +153,94 @@ def __init__(
     def revert_message(self):
         return self.message
 
+    @cached_property
+    def dev_message(self) -> Tuple[Optional[str], Optional[str]]:
+        accessor: Optional["ManagerAccessMixin"] = None
+        if self.txn:
+            accessor = self.txn
+        elif self.trace is not None:
+            self.trace, second_trace = tee(self.trace)
+            if second_trace:
+                accessor = next(second_trace, None)
+
+        if accessor is None:
+            return None, "Missing trace access."
```

Comment:
I think raising an exception may be better than having the dual string return here, it's a little confusing

Reply:
less confusing now [efb90ff](https://github.com/ApeWorX/ape/pull/1377/commits/efb90ff504aa774ea43dcfbc7fd0143200ecd7e3)